### PR TITLE
Dashboard Importer - Add Info Tile for Converted Dashboards

### PR DESCRIPTION
**Pull request by stevezease**

### CODE REVIEW

File: scripts/dashboard-importer/src/common/report.ts [hunk 1]
```diff
@@ -72,3 +74,27 @@ export function generateEmptyReport(
     date: dateString,
   };
 }
+
+// Condenses a list of warnings into a summarized format
+export function generateWarningSummary(warnings: string[], numPanels: number): string {
+  let hasCollapsibleGroupWarning = false;
+  let hasMaxTileWarning = false;
+  const warningsArr: Array<[string, number]> = warnings.reduce((acc, curr) => {
```
Comment (yqlu) [inline scripts/dashboard-importer/src/common/report.ts:82]: Can you add a comment that [string, number] is for "Warning string", priority for sorting? Maybe alias the type and comment at the type definition? It took me a bit to figure this out.

File: scripts/dashboard-importer/src/dashboards/converter/converter.ts [hunk 2]
```diff
@@ -199,6 +209,25 @@ export default class GrafanaDashboardConverter {
     };
   }
 
+  private addTroubleshootingTile() {
+    const warningSummary = `##### Generated Warnings\n\n${generateWarningSummary(this.warnings, this.panels.length)}`;
+    const troubleshootingDocsText = 'For conversion issues such as no data in charts or missing tiles, please see our [troubleshooting docs](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/master/scripts/dashboard-importer/README.md#conversion-issues)';
+    const date = new Date();
+    const dateTimeText = `Converted from ${this.fileName} on ${getDateString(date)} at ${createReportId(date)}`;
+
+    const text = `${dateTimeText}\n\n` + `${troubleshootingDocsText}\n\n` +
```
Comment (yqlu) [inline scripts/dashboard-importer/src/dashboards/converter/converter.ts:218]: Marginal preference for `[a, b, c].join('\n\n')`

Comment (stevezease) [reply scripts/dashboard-importer/src/dashboards/converter/converter.ts]: Fixed! Thank you!

File: scripts/dashboard-importer/src/common/report.ts (hunk 1 above)
Comment (stevezease) [reply scripts/dashboard-importer/src/common/report.ts]: Added a comment as well as property names for the type.
// Warnings array is an array of text, priority tuples
  const warningsArr: Array<[text: string, priority: number]>

**Review by yqlu — APPROVED** [review]

**Pull request merged**